### ISSUE TITLE
New Exchange: EXX

### ISSUE THREAD

**Issue by jjhesk** [ccxt/ccxt#1214]
This exchange is came from China and it is a legitimate registered company in Hong Kong. api: https://www.exx.com/help/restApi
ranked global No. 25
fee: https://www.exx.com/help/rate

Exx Group Limited

**Comment by Deathamns** [comment]
Anyone working on this?

**Comment by kroitor** [comment]
@Deathamns not yet )

**Comment by jjhesk** [comment]
@Deathamns the agent header has to be chrome. just checkedl

**Comment by kroitor** [comment]
Can set that with 'userAgent': this.userAgents['chrome'] on the exchange instance or in `describe()`.

https://github.com/ccxt/ccxt/blob/master/js/gdax.js#L18

**Comment by kroitor** [comment]
@lyn716 see this: https://github.com/ccxt/ccxt/pull/1410

**Comment by lyn716** [comment]
@kroitor I saw this before but our code should be integrated with Python module.

**Comment by kroitor** [comment]
@lyn716 in this library the python code is generated automatically from JS:
https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code

**Comment by lyn716** [comment]
@kroitor  That will be very nice.

**Comment by kroitor** [comment]
@lyn716 

Here are generated python versions from the implementation, kindly submitted by @Deathamns:

- https://gist.github.com/kroitor/5748bbafe7cac02c0c367c2a00ac78eb (sync)
- https://gist.github.com/kroitor/01532681a65dd3e4a49e5fa10bfc9375 (async)

You should be able to generate and use the same code for testing if you follow the docs literally (precisely):

1. check out #1410 
2. build it
3. install it locally
4. test it
5. report bugs here

In other words, if you follow and read everything carefully, you will find that what you're asking is already there, we just need your help testing it.

This would really-really help us merge it quickly.

**Comment by lyn716** [comment]
I got it and I will try to do this testing work ASAP by your step. 
Best wishes with ccxt, it helps our coding process too much:（

**Comment by kroitor** [comment]
@lyn716 thanks!
